CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+cmake_minimum_required( VERSION 3.16 )
+project( T8CODE DESCRIPTION "Parallel algorithms and data structures for tree-based AMR with arbitrary element shapes." LANGUAGES C CXX )
+include( CTest )
+
+option( T8CODE_ENABLE_MPI "" ON )
+option( T8CODE_BUILD_TESTS "" ON )
+option( T8CODE_BUILD_TUTORIALS "" ON )
+option( T8CODE_BUILD_EXAMPLES "" ON )
+
+if( NOT CMAKE_CXX_STANDARD )
+    set( CMAKE_CXX_STANDARD 17 )
+endif()
+
+if( NOT CMAKE_INSTALL_PREFIX )
+    set( CMAKE_INSTALL_PREFIX ${CMAKE_CURRENT_LIST_DIR}/build )
```

Comment:
In the `ccmake` configure GUI I get `[build_directory]/p4est/local`.

Reply:
Very strange, maybe p4est does some stuff with this variable when not specified by the user. However, it does the right thing when typing `cmake .. `DCMAKE_INSTALL_PREFIX=...`